ghost/core/test/e2e-api/admin/posts.test.js

```
@@ -1,5 +1,5 @@
 const should = require('should');
-const {agentProvider, fixtureManager, mockManager, matchers} = require('../../utils/e2e-framework');
+const {agentProvider, fixtureManager, mockManager, matchers, assertions} = require('../../utils/e2e-framework');
```

ESLint

[error] 'assertions' is assigned a value but never used. Allowed unused vars must match /^should$/u.

(no-unused-vars)